manifests/init.pp

```diff
@@ -240,7 +240,7 @@
   Hash $nginx_upstreams                                   = {},
   Nginx::UpstreamDefaults $nginx_upstreams_defaults       = {},
   Boolean $purge_passenger_repo                           = true,
-  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.6.0'),
+  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.16.0'),
```

Comment:
I am wondering if we can not just default to installed?

```suggestion
  String[1] $nginx_version                                = 'installed',
```

Putting an arbitrary (outdated) version does not look good, and if we do a breaking release maybe it is time to improve this?

Reply:
Fine with me, I was just trying to change as little as possible